### ISSUE TITLE
[Accessibility] Keyboard: Focus is going to the entire row in the grid, but user cannot perform any operation in  DetailsList - Grouped

### ISSUE THREAD

**Issue by venkatrgmsft** [microsoft/fluentui#13549]
**Browser Details:**
OS: Windows 10 (Version 1903 Build 18363.778)
Microsoft Edge Beta Version 81.0.416.62 (Official build) beta (64-bit)
Google Chrome Version 81.0.4044.122 (Official Build) (64-bit)

Issue Related to control : DetailsList - Grouped (https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped)

**Repro Steps :**

1)Hit the URL "https://codepen.io/venkat_codepen_12/pen/ExPPpjq".

2) Tab till a row is focused on the entire row in the grid
![image](https://user-images.githubusercontent.com/66743158/84298847-6b683d80-ab6d-11ea-8849-160918dec5bf.png)

3)Verify if user can perform any operation when focus is going to the entire row in the grid

**Actual Result: ​**

Focus is going to the entire row in the grid, but user cannot perform any operation

**Expected Result: ​**

Focus should not go to non-interactive controls. As focus is going to entire row user must be given an action to select the row by pressing enter or space

**User Impact: ​**

Keyboard users will find it confusing if focus is going to entire row, but they cannot perform any operation on them

**Comment by jdhuntington** [comment]
You can do that by arrowing up to the group header and press left/right. 
It should be possible to override the default keyboard behavior by overriding `onRenderRow` and defining your own keyboard handlers, though it might be a tricky undertaking.

**Comment by venkatrgmsft** [comment]
@jdhuntington Thanks for the response, but ideally in a detail list we can still press right , reach the expander icon and press down to reach the next row. So by default from an accessibility perspective this would the best behavior for keyboard users
1)Upon row focus when user hits the space bar it should expand the detail 
2)Upon  click on up down keys it should navigate to previous and next rows. 

The point mentioned in 1) is happening but the point mentioned in 1) is not. The default behavior should be more accessibility friendly

**Comment by xugao** [comment]
@venkatrgmsft - couple things here:
1. it's by design to have each row focusable, regardless if it can perform operations or not. That way screen reader can read out content for each row
2. for the row you pointed out, we have a bug here and it's suppose to be able to key-nav to collapse/expand. that bug is tracked in #6318
